content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+
+By default Crossplane only stores a single inactive package in the local package
+cache. 
+
+Store more inactive copies of a package with `--revision-history-limit`. 
+
+Read more about 
+[package revisions]({{< ref "../concepts/packages#configuration-revisions" >}}) 
+in the package documentation. 
+
+### xpkg login
+
+Use `xpkg login` to authenticate to the 
+[Upbound Marketplace](https://marketplace.upbound.io/) container registry.
+
+[Register with the Upbound Marketplace](https://accounts.upbound.io/register) 
+to push packages and create private repositories. 
+
+#### Flags
+
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag                            | Description                    |
+| ------------ | -------------                        | ------------------------------ |
+| `-u` | `--username=<username>`    | Username to use for authentication. | 
+| `-p` | `--password=<password>`    | Password to use for authentication. | 
+| `-t` | `--token=<token string>`   | User token string to use for authentication. | 
+| `-a` | `--account=<organization>` | Specify an Upbound organization during authentication. |
+{{< /table >}}
+
+
+#### Authentication options
+
+The `crossplane xkpg login` command can use a username and password or Upbound API token.
+
+Provide a username and password with the `--username` and `--password` flags or
+set the environmental variable `UP_USER` for a username or `UP_PASSWORD` for the
+password. 
+
+Use an Upbound user token instead of a username and password with `--token` or 
+the `UP_TOKEN` environmental variable. 
+
+{{< hint "important" >}}
+The `--token` or `UP_TOKEN` environmental variables take precedence over a 
+username and password.
+{{< /hint >}}
+
+Using `-` as the input for `--password` or `--token` hides the input.  
+For example, `crossplane xpkg login --password -`.
+
+After logging in the Crossplane CLI creates a `profile` in 
+`.crossplane/config.json` to cache unprivileged account information. 
+
+{{<hint "note" >}}
+The `session` field of `config.json` file is a session cookie identifier. 
+
+The `session` value isn't used for authentication. This isn't a `token`.
+{{< /hint >}}
+
+#### Authenticate with a registered Upbound organization
+
+Authenticate to a registered organization in the Upbound Marketplace with the 
+`--account` option, along with the username and password or token. 
+
+For example, 
+`crossplane xpkg login --account=Upbound --username=my-user --password -`.
+
+### xpkg logout
+
+Use `crossplane xpkg logout` to invalidate the current `crossplane xpkg login` 
+session.
+
+{{< hint "note" >}}
+Using `crossplane xpkg logout` removes the `session` from the 
+`~/.crossplane/config.json` file, but doesn't delete the configuration file.
+{{< /hint >}}
+
+### xpkg push
+
+Push a Crossplane package file to a package registry. 
+
+The Crossplane CLI pushes images to the 
+[Upbound Marketplace](https://marketplace.upbound.io/) at `xpkg.upbound.io` by 
+default.
```

Comment:
Should we show how to push to another registry?

Reply:
Sure, how? Can I just provide a full URI for the package argument?

Reply:
Yeah exactly - like `index.docker.io/crossplane-contrib/...`